packages/hash/api/src/model/ontology/util.ts

```diff
@@ -13,14 +13,14 @@ export const getNamespaceOfAccountOwner = async (
   params: { ownerId: string },
 ) => {
   const namespace =
-    params.ownerId === systemAccountId
+    params.ownerId === systemOrgAccountId
       ? SYSTEM_ACCOUNT_SHORTNAME
       : (
           (await UserModel.getUserById(graphApi, {
-            entityId: `${systemAccountId}%${params.ownerId}`,
+            entityId: `${systemOrgAccountId}%${params.ownerId}`,
```

Comment:
We can also use this here https://github.com/hashintel/hash/blob/88e84beca68a835394f270350bb6d9e59e7aa62b/packages/hash/subgraph/src/types/identifier.ts#L14

Reply:
Addressed in https://github.com/hashintel/hash/pull/1596/commits/6e6c358e80915bc8fb3fb132d66f3382fbfaf395